### DISCUSSION TITLE
For the select OptGroup

### DISCUSSION THREAD

**Discussion by daneyOwner** [LouisMazel/maz-ui#773] [Ideas]
Add OptGroup that is all ;)

**Comment by LouisMazel** [comment]
Hey!
What do you mean by "OptGroup"?

**Reply by daneyOwner** [reply]
Like a normal select has option groups like a category.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup

**Reply by LouisMazel** [reply]
Oh yes, nice idea ! 
I'll let you know here when it's done 

You can do a PR if you want too

**Comment by LouisMazel** [comment]
@daneyOwner 

The optgroup is now available since [v3.35.0](https://github.com/LouisMazel/maz-ui/releases/tag/v3.35.0)

Documentation: https://maz-ui.com/components/maz-select#opt-group

Example:

```vue
<template>
  <MazSelect
    v-model="optGroupValue"
    label="Select option"
    :options="optGroup"
    multiple
  />
</template>

<script setup lang="ts">
  import { ref } from 'vue'
  const selectedValue = ref()
  const optGroup = [
    { label: 'Basic colors', options: ['primary', 'secondary', 'danger'] },
    { label: 'Custom colors', options: [{ label: 'third', value: 'third' }] },
  ]
</script>
```
